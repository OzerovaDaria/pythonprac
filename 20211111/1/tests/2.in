A, B = OrderedComplex(1, 1), OrderedComplexMul(1, 23)
print(B<A, B@B)
