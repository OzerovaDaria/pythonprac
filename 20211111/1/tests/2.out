False 530.0
